src/main/java/net/mcreator/preferences/data/HiddenSection.java

```diff
@@ -37,10 +34,12 @@ public class HiddenSection extends PreferencesSection {
 	public IntegerEntry projectTreeSplitPos;
 	public BooleanEntry workspaceSortAscending;
 	public PreferencesEntry<SortType> workspaceSortType;
-	public PreferencesEntry<File> java_home;
+	public FileEntry java_home;
 	public StringEntry uiTheme;
 	public BooleanEntry enableJavaPlugins;
 
+	public FileEntry lastWorkspace;

```

Comment:
Last workspace is stored in WorkspaceSelector#getRecentWorkspaces()#getList()#get(0) so this entry is not needed

Reply:
Last opened